src/commons/services/utils.js

```diff
@@ -96,6 +96,24 @@
             });
         };
 
+        vm.importCluster = function(uri, data) {
+            var url, actionRequest, request;
+
+            url = config.baseUrl + uri;
+
+            actionRequest = {
```

Comment:
Utils.js will have file contains get and post request many times, So can we create two different functions. One for creating a Get request and another for Post request. So we done have to write things again and again.